### ISSUE TITLE
Scraping genres on 1pondo videos

### ISSUE THREAD

**Issue by NapInFever** [DoctorD1501/JAVMovieScraper#47]
I've noticed that when I use 1pondo scraper it doesn't scrape genres. I suppose it can be explained by the fact that you don't observe tags on the video page. Nevertheless there is a possibility to filter videos by genres on 1pondo website. Is it possible to use this feature to scrape genres for videos?

**Comment by DoctorD1501** [comment]
Are the genres on the page of the actual movie itself? I don't see them anywhere, but perhaps I'm just missing it. The filtering you can see on the site is not helpful because I would have to scrape every single page of every single genre (might be hundreds of pages) looking for the movie you are currently on to see if that genre applies to it. While it's possible to do this, it would be super slow to scrape.

**Comment by NapInFever** [comment]
No, there are no genres on the movie page. What a sloppy tags system they designed. Perhaps we might wait till their site improve or use some JAV db to tag them. BTW I am curious is it possible to scrape every single page of every single genre and then assign genres to every single video with the result being written in some file? So the JavMovieScraper could refer to this file for genres instead of scraping again. And if you want to update this file with recent titles you rescrape every genre again.

**Comment by DoctorD1501** [comment]
It might be possible to write something like that, but it would be fairly complicated to write. I'm going to leave this issue open with a "Help Wanted" tag in case someone else wants to take a stab at it.
